pyocd/target/builtin/__init__.py

```diff
@@ -242,14 +243,18 @@
           'm263kiaae' : target_M261.M263KIAAE,
           'm487jidae' : target_M480.M487JIDAE,
           'm2354kjfae' : target_M2354.M2354KJFAE,
+          'hc32f451xc' : target_HC32F45x.HC32F451xC,
+          'hc32f451xe' : target_HC32F45x.HC32F451xE,
+          'hc32f452xc' : target_HC32F45x.HC32F452xC,
+          'hc32f452xe' : target_HC32F45x.HC32F452xE,
           'hc32f460xc' : target_HC32F460.HC32F460xC,
           'hc32f460xe' : target_HC32F460.HC32F460xE,
           'hc32f4a0xg' : target_HC32F4A0.HC32F4A0xG,
           'hc32f4a0xi' : target_HC32F4A0.HC32F4A0xI,
           'hc32m423xa' : target_HC32M423.HC32M423xA,
           'hc32f120x6' : target_HC32x120.HC32F120x6TA,
           'hc32f120x8' : target_HC32x120.HC32F120x8TA,
-          'hc32m120' : target_HC32x120.HC32M120,
+          'hc32m120x6' : target_HC32x120.HC32M120,
```

Comment:
Should `hc32m120` be deprecated but kept for backwards compatibility?

Reply:
Thanks. I'll keep hc32m120.